main.js

```diff
@@ -0,0 +1,102 @@
+const Game = function () {
+  let level = 0;
+  let time = 3;
+  let remaining = 0;
+  let timeVar;
+  let timeVare;
+  let isWin = true;
+
+  const randomColor = function () {
+    const r = Math.floor(Math.random() * Math.floor(255));
+    const b = Math.floor(Math.random() * Math.floor(255));
+    const g = Math.floor(Math.random() * Math.floor(255));
+    if (r == 0 && b == 0 && g == 128) {
```

Comment:
do not use `==` in JS we use `===` and `!==`

here is the reason: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a